samples/infiniteStreaming.js

```diff
@@ -0,0 +1,284 @@
+/**
+ * Copyright 2019 Google LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * This application demonstrates how to perform infinite streaming using the
+ * streamingRecognize operation with the Google Cloud Speech API.
+ * Before the streaming time limit is met, the program uses the
+ * 'result end time' parameter to calculate the last 'isFinal' transcription.
+ * When the time limit is met, the unfinalized audio from the previous session
+ * is resent all at once to the API, before continuing the real-time stream
+ * and resetting the clock, so the process can repeat.
+ * Incoming audio should not be dropped / lost during reset, and context from
+ * previous sessions should be maintained as long the utterance returns an
+ * isFinal response before 2 * streamingLimit has expired.
+ * The output text is color-coded:
+ *    red - unfinalized transcript
+ *    green - finalized transcript
+ *    yellow/orange - API request restarted
+ */
+
+'use strict';
+
+/**
+ * Note: Correct microphone settings required: check enclosed link, and make
+ * sure the following conditions are met:
+ * 1. SoX must be installed and available in your $PATH- it can be found here:
+ * http://sox.sourceforge.net/
+ * 2. Microphone must be working
+ * 3. Encoding, sampleRateHertz, and # of channels must match header of audioInput file you're
+ * recording to.
+ * 4. Get Node-Record-lpcm16 https://www.npmjs.com/package/node-record-lpcm16
+ * More Info: https://cloud.google.com/speech-to-text/docs/streaming-recognize
+ * 5. Set streamingLimit in ms. 10000 ms = 10 seconds. Maximum limit should be 1/2 of SpeechAPI Streaming Limit.
+ */
+
+function infiniteStream(
+  encoding,
+  sampleRateHertz,
+  languageCode,
+  streamingLimit
+) {
+  // [START speech_transcribe_infinite_streaming]
+
+  //const encoding = 'LINEAR16';
```

Comment:
Throughout the changes, could you leave a space between the `//` and the first character of the comment?